### ISSUE TITLE
npm install dependencies failed (verison 6.0.0)

### ISSUE THREAD

**Issue by smalldice** [tcoopman/image-webpack-loader#198]
<img width="1071" alt="WechatIMG9" src="https://user-images.githubusercontent.com/17645579/68097514-66198500-fef2-11e9-9363-992ce4ed78f1.png">

**Comment by anikethsaha** [comment]
Can you share more details with version and commands .

**Comment by NigeAtAnd** [comment]
Possibly related, I've started using this loader today and get the following error on Gitlab CI:

```
Error: Cannot find module 'imagemin-gifsicle'
Require stack:
- /frontend/node_modules/image-webpack-loader/index.js
...
```

It works fine on my local box.

**Comment by anikethsaha** [comment]
do you have `imagemin-gifsicle` in your devdep or dep ?

**Comment by NigeAtAnd** [comment]
No, I thought this loader came bundled with the various imagemin plugins?

**Comment by anikethsaha** [comment]
Actually this loader has them in `optionalDependencies` so it never 100% assured that they will be installed if it's in `optionalDependencies`. Its recommended to install them as devDep
